content/tbb/how-to-verify-signature/contents.lr

```diff
@@ -70,7 +70,60 @@ The examples below assume that you downloaded these two files to your "Downloads
 
 #### For Windows users:
 
-    gpgv --keyring .\tor.keyring Downloads\torbrowser-install-win64-9.0_en-US.exe.asc Downloads\torbrowser-install-win64-9.0_en-US.exe
+    gpgv --keyring .\tor.keyring Downloads\torbrowser-install-win64-9.0_en-US.exe.asc.txt Downloads\torbrowser-install-win64-9.0_en-US.exe
+	
+#### Alternative for windows
+
+After installing GnuPG open cmd.exe and follow the steps below
+
+	* gpg.exe --keyserver pool.sks-keyservers.net --recv-keys 0x4E2C6E8793298290
```

Comment:
We removed keyserver and switched to WKD after Tor Browser OpenPGP key was attacked by a signature flood. These instructions are incorrect. Read more about the attack here: https://lists.torproject.org/pipermail/tor-project/2019-July/002402.html